### ISSUE TITLE
Ability to set custom <canvas> width and height for better quality

### ISSUE THREAD

**Issue by AVargas89** [hrynko/vue-pdf-embed#1]
Hi,
Great component by the way!
Are you able to implement props that will allow custom width/height for the canvas? 

We are losing some quality appearance with just the client viewport dimensions being passed.

Thank you.

**Comment by hrynko** [comment]
Hi @AVargas89,

Thank you, I will try to look into it later today or tomorrow.

**Comment by hrynko** [comment]
It seems to me that the loss of quality is caused by the scaling of the viewport, so setting a custom canvas width/height doesn't help. Currently, the canvas element inherits the size of its parent, and I find this approach more flexible than setting a specific canvas size with the props. You can set the size of the wrapper as follows, and `vue-pdf-embed` will adjust to fit:

```vue
<div :style="{ width: '320px' }">
  <vue-pdf-embed />
</div>
```

I see your point about the loss of quality and will prepare a release with a fix, however I wouldn't add width/height props.

**Comment by AVargas89** [comment]
Thank you! This solution worked for now :)

**Comment by hrynko** [comment]
Anytime! New ideas and issues are welcome.

By the way, the quality loss fix has been released in `1.0.2`
